plugins/checklist.md

```diff
@@ -11,9 +11,9 @@ controls: toolbar button
 
 The **Checklist** plugin helps the user keep track of all required actions by creating a simple, clean, and easy to use TODO list.
 
-## Format
+## Live example
```

Comment:
Should this say "Try our Checklist demo"? As that's what I've seen used elsewhere in this PR.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d